pavelib/assets.py

```diff
@@ -734,12 +734,10 @@ def webpack(options):
     Run a Webpack build.
     """
     settings = getattr(options, 'settings', Env.DEVSTACK_SETTINGS)
-    environment = 'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
+    sh(cmd('NODE_ENV={node_env} $(npm bin)/webpack --watch --watch-poll=200 --output-path={static_root_cms}'.format(
```

Comment:
The key here is that we need to specify `--output-path`. Normally, webpack outputs to a totally different directory, then `collect_assets` moves those files to the correct location. We need to cut out the middleman for correct watcher behavior.

Reply:
The changes in this file are more "figure out what's going on" than "suggest a fix".